test/desktop-lite/test_xtigervnc_novnc_started.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Check if xtigervnc & noVnc processes are running after successful installation and initialization
+check_process_running() {
+    port=$1
+    # Get process id of process running on specific port
+    PID=$(lsof -i :$port | awk 'NR==2 {print $2}')
+    if [ -n "$PID" ]; then
+        CMD=$(ps -p $PID -o cmd --no-headers)
+        GREEN='\033[0;32m'; NC='\033[0m'; RED='\033[0;31m'; YELLOW='\033[0;33m';
+        echo -e "${GREEN}Command running on port $port: ${YELLOW}$CMD${NC}"
+    else
+        echo -e "${RED}No process found listening on port $port.${NC}"
+    fi
+}
+
+check "Whether xtigervnc is Running" check_process_running 5901
```

Comment:
Why do we see `[2024-08-14T08:15:13.432Z] /usr/local/share/desktop-init.sh: line 102: exec: : not found` ?

https://github.com/devcontainers/features/actions/runs/10383896871/job/28749788841?pr=1090#step:4:3972

Reply:
it is because of the file `desktop-init.sh` receiving no arguments and it's set as an `entrypoint` file to be executed, if you can see line 102: of `desktop-init.sh` then `exec` command fails because of no arguments being provided to it, 
```
log "Executing \"\$@\"."
    exec "$@"
```

if I will change the `desktop-init.sh` file's contents to :
```
# Run whatever was passed in
if [ -n "$1" ]; then
    log "Executing \"\$@\"."
    exec "$@"
else
    log "No command provided to execute."
fi
```
then the log looks like this now:
```
root@a58ce4f02d73:/# cat /tmp/container-init.log
[Wed Aug 14 09:35:31 PM UTC 2024] ** SCRIPT START **
[Wed Aug 14 09:35:31 PM UTC 2024] Running "/etc/init.d/dbus start".
[Wed Aug 14 09:35:31 PM UTC 2024] Starting Xtigervnc.
[Wed Aug 14 09:35:32 PM UTC 2024] Xtigervnc started.
[Wed Aug 14 09:35:32 PM UTC 2024] noVNC started.
[Wed Aug 14 09:35:32 PM UTC 2024] No command provided to execute.
[Wed Aug 14 09:35:32 PM UTC 2024] ** SCRIPT EXIT **
```

Reply:
Great, thanks!